week3/menu-recommend/src/components/Header/index.jsx

```diff
@@ -0,0 +1,21 @@
+import * as S from './style';
+
+const Header = ({ selectOption, selectOptionHandler, startPickHandler }) => {
+  return (
+    <S.HeaderStyle>
+      <h1>🍛 오늘의 점메추 🍛</h1>
+      {selectOption !== '' && (
```

Comment:
```suggestion
      {selectOption && (
```
이렇게만 해도 되겠다! 빈 문자열은 falsy값, 빈 문자열이 아니면 truthy 값이라서 가능합니당